home/templates/home/content_page.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags %}
+
+{% block body_class %}template-contentpage{% endblock %}
+
+
+{% block content %}
+    <h1>{{ page.title }}</h1>
+    <p class="meta">{{ page.date }}</p>
+
+    <div class="intro">{{ page.intro }}</div>
```

Comment:
I don't think we have `date` and `intro` fields in our `ContentPage` model. We do have `title`, `subtitle`, and `body` though (at least for the web section of the page)